src/ui/Typeahead.js

```diff
@@ -64,7 +73,6 @@ Typeahead.propTypes = {
 
 const typeaheadDefaultProps = {
   options: [],
-  labelKey: (item) => item,
```

Comment:
I'm not sure if this should be deleted

Reply:
I added it again, must be a false warning in PHPStorm